### ISSUE TITLE
Add Ability To Customize Messages In Status Check

### ISSUE THREAD

**Issue by ndibari-etsy** [palantir/policy-bot#292]
Hi there! I've been trying out `policy-bot` for my team and it's been working really well. One thing I think would be cool would be to customize the messages in the status check for pending or approved pull requests. The messages could be specified in the server config file and the `Evaluate` method can lookup the message from a `MessageConfig` struct (with fallbacks to sensible defaults if there are no messages supplied in the config.)

Additionally, my team was also thinking about adding internationalization to the message strings using [go-i18n](https://github.com/nicksnyder/go-i18n) to add support for other languages to policy-bot. Please let me know if this is something you might be interested in.

Thank you for making policy-bot!

**Comment by asvoboda** [comment]
Hey @ndibari-etsy,

I think having a way to customise the evaluation status checks makes sense. One caveat is that generally these need to be pretty short (140 characters) to avoid validation errors, though the [GitHub docs](https://docs.github.com/en/rest/reference/repos#create-a-commit-status) on the topic are sparse.

On the i18n work, I'd defer the question to @bluekeyes. Perhaps we should track via a separate issue?

**Comment by bluekeyes** [comment]
Yeah, customizing the status messages seems reasonable. I'm curious to see an example of the changes you'd like to make. Is there something we could change in the default messages to make them more useful?

For the i18n changes, I think I'd need to see an example of the changes for this project or see a similar project that is already using `go-i18n`. While I agree that it would be ideal if policy-bot supported multiple languages, I'm worried that this could add a large maintenance burden for our team and that we wouldn't be able to support it long-term.

**Comment by ndibari-etsy** [comment]
We're experimenting with what language to use in the case that `policy-bot` blocks a PR (to be in line with what other teams are used to with their workflows.) We can share back what we've found, if it feels less useful as the default, we can look to contribute a change that makes this configurable.

As for how we would implement this, I was looking at the `frontend.go` file as inspiration. There a `FilesConfig` is defined that reads the directories for static files and templates here: https://github.com/palantir/policy-bot/blob/develop/server/handler/frontend.go#L29-L37 and then later on when the code needs the value of the variable it either reads it from the config or falls back to the default: https://github.com/palantir/policy-bot/blob/develop/server/handler/frontend.go#L69-L76. I was thinking of doing something similar, although since the evaluation happens in a different section I'm not sure if the pattern could be reused there. Do you think this would make sense as an option?
